### PR TITLE
Update Readme for 1.0

### PR DESCRIPTION
**Pull request by andersio**

### CODE REVIEW

**Comment by andersio** [comment]
Thanks for catching these. 😸

File: README.md [hunk 1]
```diff
@@ -1,45 +1,14 @@
...
+#### Streams of values over time. Tailored for Swift.
 
-ReactiveSwift is a Swift framework inspired by [Functional Reactive Programming](https://en.wikipedia.org/wiki/Functional_reactive_programming). It provides APIs for composing and transforming **streams of values over time**.
+[![Carthage compatible](https://img.shields.io/badge/Carthage-compatible-4BC51D.svg?style=flat)](#carthage) [![CocoaPods compatible](https://img.shields.io/cocoapods/v/ReactiveSwift.svg)](#cocoapods) [![SwiftPM compatible](https://img.shields.io/badge/SwiftPM-compatible-orange.svg)](#swift-package-manager) [![GitHub release](https://img.shields.io/github/release/ReactiveCocoa/ReactiveCocoa.svg)](https://github.com/ReactiveCocoa/ReactiveCocoa/releases) ![Swift 3.0.x](https://img.shields.io/badge/Swift-3.0.x-orange.svg) ![platforms](https://img.shields.io/badge/platform-iOS%20%7C%20macOS%20%7C%20tvOS%20%7C%20watchOS%20%7C%20Linux-lightgrey.svg)
```
Comment (mdiep) [inline README.md]: The release shield still points to RAC. 😳 

Can you update that?

File: README.md [hunk 2]
```diff
@@ -1,45 +1,14 @@
...
 
- 1. [Introduction](#introduction)
- 1. [Example: online search](#example-online-search)
- 1. [How does ReactiveSwift relate to Rx?](#how-does-reactiveswift-relate-to-rx)
- 1. [Getting started](#getting-started)
- 1.  [Playground](#playground)
+⚠️ [Looking for the Objective-C API?][] ⚠️ [Still using Swift 2.x?][]
```
Comment (mdiep) [inline README.md]: I don't think we need a link to the Obj-C API here. There's no reason for someone to end up here when looking for the Obj-C APIs.

File: README.md [hunk 3]
```diff
@@ -55,67 +24,78 @@ code and state to bridge the gap.
 For more information about the concepts in ReactiveSwift, see the [Framework
 Overview][].
 
+## Looking for Cocoa extensions?
```
Comment (mdiep) [inline README.md]: This could probably be up by the other _looking for …?_ links.

File: README.md [hunk 4]
```diff
@@ -196,54 +174,37 @@ Due to its nature, a stream's stack trace might have dozens of frames, which, mo
...
 
 ## How does ReactiveSwift relate to Rx?
 
-ReactiveCocoa was originally inspired, and therefore heavily influenced, by
-Microsoft’s [Reactive
-Extensions](https://msdn.microsoft.com/en-us/data/gg577609.aspx) (Rx) library. There are many ports of Rx, including [RxSwift](https://github.com/ReactiveX/RxSwift), but ReactiveCocoa is _intentionally_ not a direct port.
-
-**Where ReactiveSwift differs from Rx**, it is usually to:
+While ReactiveCocoa was inspired and heavily influenced by [ReactiveX][] (Rx), ReactiveSwift is
+an opinionated implementation of the grand concept, and _intentionally_ not a direct port like
```
Comment (mdiep) [inline README.md]: _the grand concept_ reads a little weird. I'd take out the word _grand_ or write something like _is an opinionated implementation of reactive programming_.

File: README.md [hunk 5]
```diff
@@ -196,54 +174,37 @@ Due to its nature, a stream's stack trace might have dozens of frames, which, mo
...
+RxSwift.
```
Comment (mdiep) [inline README.md]: We should continue to link to RxSwift, I think.

**Comment by andersio** [comment]
@mdiep Comments addressed. 😸

**Pull request merged**